src/pages/Checkout/CheckoutPrice.js

```diff
@@ -69,7 +76,7 @@ export function CheckoutPrice({ setMsg }) {
     if (address.length === 0) {
       toast.error("Please Add Address");
       setTimeout(() => {
-        navigate("/userProfile");
+        navigate("/user_profile");
```

Comment:
Avoid using settimeout this way... instead you can use a async function and redirect